v1/pipeline/common/number-variable.yaml

```diff
@@ -19,6 +19,9 @@ allOf:
             format: double
           - type: string
             pattern: ((^[+-]?[0-9]*\.?[0-9]+$)|(<\+.+>.*))
+        metadata:
+          inputProperties:
+            dependsOn: [ ./type ]
```

Comment:
`./` is redundant here.  `./type` is equivalent to `type`. Shall we change it to `type` that would look simpler. And anyone following this PR would not add `./` unnecessarily.